test/runnable/mars1.d

```diff
@@ -1483,41 +1483,6 @@ void test4()
     assert(binPow2() == 1.0L);
 }
 
-////////////////////////////////////////////////////////////////////////
-// https://issues.dlang.org/show_bug.cgi?id=13474
-
-
-double sumKBN(double s = 0.0)
-{
-    import std.math : fabs;
```

Comment:
This is an intrinsic: https://github.com/dlang/druntime/blob/74b5ba165abf01eaa3ba7244d64fdcf3c2147827/src/core/math.d#L129-L138